modules/steamos/sysctl.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, ... }:
+
+let
+  inherit (lib)
+    mkDefault
+    mkIf
+    mkMerge
+    mkOption
+    types
+  ;
+  cfg = config.jovian.steamos;
+  mkJovianDefault = lib.mkOverride 999;
+in
+{
+  options = {
+    jovian.steamos = {
+      enableSysctlConfig = mkOption {
```

Comment:
Should `useSteamOSConfiguration` and `enableSysctlConfig` be consistent in choosing `Config` or `Configuration`?

Reply:
Surveyed with `grep -i 'config.*mkoption'` and `Config` is preferred.